src/fundamental/efi-fundamental.h

```diff
@@ -1,7 +1,7 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 #pragma once
 
-#if !SD_BOOT
+#if !defined(SD_BOOT_EFI_H_INCLUDED)
```

Comment:
can the includes be rearranged so that this is not necessary?

Reply:
I don't think so, it's not about the order really. 
`efi.h` and `efi-fundamental.h` are both defining these structures (and this is not the only situation when this happens) and both are included in `chid.c`, so compilation fails with SD_BOOT=0.

I guess we can remove these definitions from `efi.h` and only keep them in `efi-fundamental.h`.

Reply:
yeah de-duplicating sounds fine